prediction-applier-extension/src/main/java/org/alfresco/hxi_connector/prediction_applier/service/PredictionServiceImpl.java

```diff
@@ -160,4 +165,34 @@ public List<String> getPredictedProperties(NodeRef nodeRef)
                 .map(qName -> qName.toPrefixString(namespaceService))
                 .collect(toList());
     }
+
+    @Override
+    public void reviewPrediction(NodeRef predictionNodeRef, ReviewStatus reviewStatus) throws PredictionStateChangedException
+    {
+        ChildAssociationRef parentAssocRef = nodeService.getPrimaryParent(predictionNodeRef);
+        NodeRef parentNode = parentAssocRef.getParentRef();
+        Map<QName, Serializable> existingProperties = new HashMap<>(nodeService.getProperties(parentNode));
```

Comment:
I think it would be more efficient to work with a single property here (assuming the API only allows confirming/rejecting a single property).

Reply:
Good point, changed